setup.py

```diff
@@ -10,13 +10,12 @@
     package_dir={'eb_sqs': 'eb_sqs'},
```

Comment:
we still need to bump the version manually here, so we'll be able to consume the change

Reply:
ok.